main.go

```diff
@@ -198,16 +204,29 @@ func ExtractAllServiceNames(firstResultsCosts map[string]float64, secondResultsC
 	return allServiceNames
 }
 
-func GetCosts(svc *costexplorer.CostExplorer, start string, end string, costmetric string, grouping string, serviceFilter string) map[string]float64 {
-	var filter *costexplorer.Expression
+func GetCosts(svc *costexplorer.CostExplorer, start string, end string, costmetric string, grouping string, serviceFilter string, tagFilters []string) map[string]float64 {
+	// Assemble filters
```

Comment:
The diff is confusing here, but I am creating an `Expression` for any tags specified, and for the service if specified. Then at the end if there's more than one expression I AND them together, or if there's just one I set filter to that.